boards/nxp/frdm_rw612/frdm_rw612_defconfig

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright 2022 NXP
```

Comment:
Is this copyright accurate? I guess the file is copied from the RD-RW612-BGA, which is copyright 2022?

Reply:
yes